compiler/testData/codegen/box/sam/kt52417.kt

```diff
@@ -0,0 +1,16 @@
+// TARGET_BACKEND: JVM
+// WITH_RUNTIME
+// WITH_STDLIB
+abstract class TypeToken<T>
```

Comment:
Please add `package test` here, otherwise the test fails if run as a part of `compiler/android-tests` (explicit package directive helps since that configuration replaces package in each test with a unique name by regexp, and runs all tests in a single project)